database/schema.sql

```diff
@@ -171,6 +171,7 @@ create table Orders
 create table Reviews
 (
     oid         integer primary key not null references Orders (id) on delete cascade,
+    cid         varchar(20) references Customers (id),
```

Comment:
This is not necessary since each cid is uniquely linked to one cid

Reply:
Did it this way so that easier to retrieve cid for each review when we display. Rather than using queries.

Reply:
Unfortunately this creates redundant attributes in the schema, which I think should be avoided in the context of 2102 unless there are strong justifications (e.g. frequent read access)

Reply:
Alright. Got ur point. Will do